File: Project/MyProject/MyProject/Source/MyProject/Private/BasicFrame/Libs/MsgRoute/MsgCmd/SocketOpenedMR.cpp
#include "MyProject.h"
#include "SocketOpenedMR.h"
#include "MsgRouteId.h"

MY_BEGIN_NAMESPACE(MyNS)

SocketOpenedMR::SocketOpenedMR()
	: MsgRouteBase(MsgRouteId::eMRID_SocketOpened)
{

}

MY_END_NAMESPACE

File: Project/MyProject/Tool/CSToolKit/Project/ToolKit/ToolKit/Source/Libs/MsgRoute/MsgCmd/SocketOpenedMR.cs
namespace SDK.Lib
{
    public class SocketOpenedMR : MsgRouteBase
    {
        public SocketOpenedMR()
        {
            this.setMsgTypeAndsetMsgId(MsgRouteType.eMRT_BASIC, MsgRouteId.eMRID_SocketOpened);
        }
    }
}